source/code/source/playfab/QoS/XPlatSocket.cpp

```diff
@@ -20,6 +20,7 @@ namespace PlayFab
         XPlatSocket::XPlatSocket()
         {
             initialized = false;
+            memset(&timeOutVal, 0, sizeof(timeOutVal));
```

Comment:
Please double check the best way to initialize a timreOutVal (whatever it's type is).
It's not necessarily safe to just zero it like this.

Reply:
It could also fail on some platforms and not others.

Reply:
C-Style casting is the recommendation: https://stackoverflow.com/questions/6462093/reinitialize-timeval-struct